tests/test_mei.py

```diff
@@ -33,7 +33,7 @@
 
 class TestImportMEI(unittest.TestCase):
     def test_main_part_group1(self):
-        parser = MeiParser(MEI_TESTFILES[5])
+        parser = MeiParser(MEI_TESTFILES[1])
```

Comment:
Indexing files is not a nice strategy for test as this might change in the future, I would say create a variable in tests init that describes the test or file.

Reply:
I agree, but this should be done on another PR which change all tests

Reply:
Yes, I will open a relevant issue for this so we can fix it for some next release.